_infra/helm/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: case
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 11.0.0
```

Comment:
I think the chart version should start at 0.1.0. This is different to the app version, and relates to the published chart. Since this is a new chart it should start at 0.1.0 and then hit 1.0.0 when we're ready to release